onadata/apps/main/tests/test_form_gallery.py

```diff
@@ -40,6 +40,8 @@ def test_clone_with_username_and_id_string_in_uppercase(self):
 
     def test_clone_id_string_starts_with_number(self):
         self._publish_transportation_id_string_starts_with_number_form()
+        # The following lines are useless since `self.xform.id_string` does not
+        # start with numbers.
```

Comment:
ah, so `_publish_transportation_id_string_starts_with_number_form()` attempts to publish a form whose `id_string` starts with a number and (now, as of this PR) verifies that it fails. however, when it was originally written, it was intended to succeed: https://github.com/kobotoolbox/kobocat/commit/244237327e4b3173d45daf5048bf661c4b64e8e9.

Reply:
let's delete `test_clone_id_string_starts_with_number()` and rename `_publish_transportation_id_string_starts_with_number_form()` so that it starts with `test` and is executed as a unit test in its own right.